README.md

```diff
@@ -1,56 +1,82 @@
 # 🌊 Sea Lion
+
 Sea Lion makes network information available as a data source for p5.js sketches.
 
 To do that, Sea Lion relies on a local server talking to the network and serving information it collects to a JavaScript client. From there, you can use p5.js functions to consume and process the network data however you want.
 
 # Getting started
+
 1. Clone this repository
 
 ## Server
+
 For first-time setup with virtualenv:
 
 1. Open up Terminal and `$ cd server` to get to the `sealion/server/` directory
 2. `$ virtualenv venv` to create a virtualenv
 3. `$ source venv/bin/activate` to activate the virtualenv
 4. `$ pip install -r requirements.txt` to install Python dependencies
 
-Start server:
+Then either start server with GUI:
```

Comment:
I think there should be a line about what the GUI is for in case some people don't know
```suggestion
Then, start the server. You can start the Sea Lion server with a GUI (for a more user-friendly interface than Terminal):

-  `$ python src/gui.py`

or without a GUI:

-  `$ python src/main.py`
```